en/yohane.md

```diff
@@ -2,33 +2,33 @@
 title: "Yohane Commands"
 old_id: 4
 ---
-Berikut ini adalah command-command yang ada di bot Yohane.
+These are the commands for the Yohane bot.
 
 ## Yohane In-Game
 
 ### General commands
-- `!roll [angka]` - Memberikan random number
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
+- `!help` - Shows a list of all commands
+- `!subscribe pp [options]` - Notifies about the pp status such as limit after [submitting a score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options: `set`,`get`,`unset`,`toggle`)
+- `!boardshow [options]` - [Experimental] Hide yourself from the leaderboard (options: `others`,`normal`)
+- `!report` - If you find a cheater, please use this command.
```

Comment:
```suggestion
- `!report` - If you found a cheater or annoying players, use this command for report to the staff.
```

Reply:
```suggestion
- `!report` - If you encounter any cheater or an annoying player, use this command to report it to the staff.
```

Reply:
atau gini
```suggestion
- `!report` - If you encounter any cheater or an annoying player, use this command to notify our staff.
```

Reply:
You wouldn't add "annoying" in the sense of a report in a English doc, as it is too subjective

Reply:
Ah, that's fair, any suggestion on what word to use?
Maybe misbehaving?

Reply:
Well, we could describe it generally as people who break the rules or something like that

Reply:
Ah, good idea. Maybe something like this, up to you.
```suggestion
- `!report` - If you encounter any player breaking rules, use this command to notify our staff.
```